### DISCUSSION TITLE
Auto access denied option with fixed cable and RFID

### DISCUSSION THREAD

**Discussion by PETVK** [SmartEVSE/SmartEVSE-2#48] [Ideas]
Hi there,

In my configuration I use a fixed cable and a RFID reader for access. If I disconnect the cable I  also have to use my RFID tag to deny access to the charger.

I think it would be more pleasant if we only have to use the RFID tag for getting access to the charger in case of a fixed cable.
Maybe it would be an nice to have an menu option to automatic cutout access to the charger if connection to a EV is lost for more than 30 seconds or so.

Thanks for reading!

Greetings,

Peter

**Comment by mstegen** [comment]
If you set the RFID option to EnableOne, the SmartEVSE should re-lock the charger after a minute (when disconnected)

**Reply by PETVK** [reply]
Thanks, I will test it this weekend!

**Reply by PETVK** [reply]
It worked, thanks for the advise!
